server/server.go

```diff
@@ -195,8 +192,30 @@ func newServer(ctx context.Context, c Config, rotationStrategy rotationStrategy)
 		logger:                 c.Logger,
 	}
 
-	for _, conn := range c.Connectors {
-		s.connectors[conn.ID] = conn
+	// Retrieves connector objects in backend storage. This list includes the static connectors
+	// defined in the ConfigMap and dynamic connectors retrieved from the storage.
+	storageConnectors, err := c.Storage.ListConnectors()
+	if err != nil {
+		return nil, fmt.Errorf("server: failed to list connector objects from storage: %v", err)
+	}
+
+	if c.EnablePasswordDB {
+		// The passwordDB connector is an internal connector maintained by the server.
+		s.mu.Lock()
+		s.connectors["local"] = Connector{
```

Comment:
This needs to be part of the static connectors, or you won't include it when you list all connectors.